### PR TITLE
Update test-status.md

### PR DESCRIPTION
**Pull request by yavgel**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/test-results/test-status.md [hunk 1]
```diff
@@ -47,6 +47,28 @@ values={[
 ]}>
 
 <TabItem value="java">
+    
+* JUnit 4
+```java reference title="Test Reporting with JUnit 4 Test Watcher"
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/test-results/test-status.md:52]: @titusfortner @nikolay-advolodkin @diemol, we've received this Pull Request recommending we add a JUnit 4 example in addition to the existing ones (JUnit 5 and TestNG). Can you please take a look?

**Review by titusfortner — CHANGES_REQUESTED** [review]
Use this as the reference to match the others
https://github.com/saucelabs-training/demo-java/blob/docs-1.0/selenium-junit4-examples/src/test/java/com/saucedemo/selenium/junit4/demo/SeleniumTest.java#L55-L70

We can also created nested tabs for this if that's easier to read.

File: docs/test-results/test-status.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lauramthomas) [reply docs/test-results/test-status.md:52]: @titusfortner @nikolay-advolodkin Following up on this open PR from April. Can you please check the JUnit 4 code sample added above for accuracy?

[the remark this answers is not in the document]
Comment (nadvolod) [reply docs/test-results/test-status.md:52]: @yavgel thanks so much for the excellent suggestion. The code is definitely correct. However, we don't reference code directly as it's subject to change and break. Instead we reference a release of the docs. In this case, this is the correct link to put 
https://github.com/saucelabs-training/demo-java/blob/a4f257e84c54897e20aa05ebe3d6d32a88755221/selenium-junit4-examples/src/test/java/com/saucedemo/selenium/junit4/demo/SeleniumTest.java#L55-L70

@lauramthomas if you just replace the code above with the link that I shared, similar to how [Junit5 reference code](https://github.com/saucelabs/sauce-docs/blob/659f5d75f80e5ee8b20494cf38b90d2ef34c0198/docs/test-results/test-status.md). Then this is correct.

**Review by nadvolod — CHANGES_REQUESTED** [review]

**Comment by lysannep** [comment]
@aliciakrebs Can you help with this PR?

**Pull request closed without merging**